src/resourcesManager.ts

```diff
@@ -31,6 +31,7 @@ export class LoadedResources {
   // User data (specific to current resourcepack/version)
   customBlockStates?: Record<string, any>
   customModels?: Record<string, any>
+  customItemModelData: Record<string, any[]> = {}
```

Comment:
I think it would be useful in future to actually have original json files for further parsing. Let's rename for now to something like customItemModelNames: Record<string, string[]> (as I understand it's the right type here)

Reply:
It maps custom model data IDs to the model names so I'm not sure that's a good name either. :S I actually did struggle naming this. lol